css/mirador.css

```diff
@@ -1231,6 +1231,52 @@ li.highlight {
   background-repeat: no-repeat;
 }
 
+.mirador-line-type {
+  border-radius: 5px !important;
+  padding: 0px !important;
+  width: 55px !important;
+  opacity: 0.6;
+  border: 1px solid #91765d;
+  background: white none repeat scroll 0 0;
+}
+
+.mirador-line-type ul {
+  color: #333333;
+  margin-top: 0px !important;
+}
+
+.mirador-line-type li {
+  text-shadow: none !important;
+  font-size: 10px !important;
+}
+
+.mirador-line-type>i {
+  padding-left: 35px;
+  margin: 4px 0px 0px 4px !important;
+}
+
+.mirador-line-type i {
+  color: #333333;
+  width: 55px;
+  height: 10px;
+  font-size: 15px !important;
+  background-repeat: no-repeat;
+}
+/*
+ TODO find font-icons for border types
+*/
+.mirador-line-type .solid {
+  background-image: url('../images/border_type_1.png');
```

Comment:
mirador.css can't use `url('../images` because the location of the `images` folder can be set by `buildPath` and `imagesPath`.  You need to do something like this: https://github.com/IIIF/mirador/blob/release2.1/js/src/viewer.js#L60 to get those paths and set the `background-image` in the JS, not the CSS